internal/common/function/function.go

```diff
@@ -149,8 +261,6 @@ func CheckVeleroBackupLabels(backup *velerov1api.Backup) bool {
 	return exists && value == constant.ManagedByLabelValue
 }
 
-// TODO not used
```

Comment:
why TODO was removed? run `grep -Inr 'GetNonAdminBackupFromVeleroBackup' .` this function is not used anywhere

Reply:
I will remove that function from function.go and use one from common_nab, that was left over from my refactor which I forgot to delete.

Reply:
Ok this should not yet been removed. It's unused, but we will need that functionality.

Reply:
please undo removal

Reply:
done.